### ISSUE TITLE
Quick Start doesn't work

### ISSUE THREAD

**Issue by edtan** [concourse/concourse#3758]
## Bug Report

The Quick Start doesn't work.  Running the commands results in
```
the required flag `--worker-tsa-worker-private-key' was not specified
```

## Steps to Reproduce

Follow the Quick Start.

## Expected Results

It should start.

## Actual Results

It doesn't.

## Additional Context

The TSA worker private key was made required in 1273f54c.  It seems you can work around this by faking a non-existent private-key file path in order to let `quickstart` generate one automatically.  To do this, add this to the end of the `docker-compose.yml` file as part of the environment variables:
```
CONCOURSE_WORKER_TSA_WORKER_PRIVATE_KEY: a
```
## Version Info

* Concourse version: 5.1.0
* Deployment type (BOSH/Docker/binary): docker-compose
* Infrastructure/IaaS:
* Browser (if applicable):
* Did this used to work?

**Comment by xtremerui** [comment]
Thx for submitting the issue!

@ddadlani

**Comment by xtremerui** [comment]
https://github.com/concourse/concourse/pull/3766 should be a quick fix.

**Comment by ddadlani** [comment]
Thanks for picking this up, @xtremerui!

**Comment by kdthanvi** [comment]
I think this issue is still not fixed.
```
P32149:concourse-ci karan.thanvi$ docker-compose ps
           Name                          Command               State     Ports  
--------------------------------------------------------------------------------
concourse-ci_concourse-db_1   docker-entrypoint.sh postgres    Up       5432/tcp
concourse-ci_concourse_1      dumb-init /usr/local/conco ...   Exit 1           
P32149:concourse-ci karan.thanvi$ 
P32149:concourse-ci karan.thanvi$ 
P32149:concourse-ci karan.thanvi$ docker logs concourse-ci_concourse_1 
the required flag `--worker-tsa-worker-private-key' was not specified
P32149:concourse-ci karan.thanvi$ 
```
Can anyone help ?

**Comment by cirocosta** [comment]
Hi @kdthanvi, 

Sorry for giving the impression that this was fixed **and released**, my bad.

This fix will land in the next `concourse/concourse` that gets released - 5.2 (to be released very very soon).

To verify that the release-candidates for 5.2 already have the fix in, try changing the `image` in the `docker-compose` from `concourse/concourse` to `concourse/concourse-rc`.

```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index b4242b9..8be012b 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -10,7 +10,7 @@ services:
       PGDATA: /database
 
   concourse:
-    image: concourse/concourse
+    image: concourse/concourse-rc
     command: quickstart
     privileged: true
     depends_on: [concourse-db]
```

Thanks!
